build.psm1

```diff
@@ -1012,7 +1012,7 @@ function Start-PSPester {
     # All concatenated commands/arguments are suffixed with the delimiter (space)
```

Comment:
@JamesWTruher, your last commit had 2 failures in `PowerShell-CI-windows`
WindowStyle argument.-WindowStyle Maximized should work on Windows
```powershell
Expected exactly 'Maximized', but got Hidden.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\ConsoleHost.Tests.ps1: line 839
839:             $showCmd | Should -BeExactly $WindowStyle
```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=.Xml.XDocument.dll; SideIndicator=<=}.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 121
121:         $diffs | Should -BeExactly $null
```